ui/src/components/PathList/PathListItem.tsx

```diff
@@ -0,0 +1,129 @@
+                return "delete"
+            default:
+                return "info"
+        }
+    }
+
+    const getFromTypeColor = (type: string) => {
+        switch (type) {
+            case "create":
+                return "create"
+            case "update":
+                return "delete"
+            case "delete":
+                return "delete"
+            default:
+                return "info"
+        }
+    }
+
+    return (
+        <Accordion expanded={!accordions.find(x => x.id === JSON.stringify(change))?.isCollpased}>
+            <AccordionSummary
+                expandIcon={<ExpandMoreIcon />}
+                aria-controls="panel2a-content" onClick={() => onClick(JSON.stringify(change))}>
+                <div className='accordionTitle'>
+                    <div className='path'>
+                        <span className='pathPrefix'>{change.Value.Key}</span>&nbsp;
+                        <span className='pathName'>{change.Key}</span>
+                    </div>
+                    <div>
+                        <span className='change total'>Changes: {changeVal.TotalChanges}</span>&nbsp;
+                        {changeVal.CreatedChanges > 0 && <span className='change create'>Created: {changeVal.CreatedChanges}</span>}
+                        {changeVal.UpdatedChanges > 0 && <span className='change update'>Updated: {changeVal.UpdatedChanges}</span>}
+                        {changeVal.DeletedChanges > 0 && <span className='change delete'>Deleted: {changeVal.DeletedChanges}</span>}
+                    </div>
+                </div>
+            </AccordionSummary>
+            <AccordionDetails>
+                <div className={`${filteredChanges[0]?.Changelog?.type} changeCategory`}>{filteredChanges[0]?.Changelog?.type + "d"}</div>
+                {filteredChanges?.map((path: any) => {
+                    return (<Accordion key={JSON.stringify(path)} expanded={(() => isExpand(path))()}>
+                        <AccordionSummary
+                            onClick={() => onClick(JSON.stringify(path))}
+                            expandIcon={<ExpandMoreIcon />}
+                            aria-controls="panel2a-content">
+                            <div>
+                                <span className={`operation ${path.Operation}`}>{path.Operation}</span>&nbsp;
+                                <span className='pathName'>{path.Changelog.path.join(" ")}</span>
+                            </div>
+                        </AccordionSummary>
+                        <AccordionDetails>
+                            <span>Path:</span>
+                            {path.Changelog.path.slice(1).map((path: any, index: number) =>
+                                <div key={`${path + index}`} style={{ paddingLeft: `${(index + 1 * 0.4)}em` }}>{path}</div>)
+                            }
+                            <div style={{ marginTop: "10px" }}>
+                                <Grid container spacing={2}>
```

Comment:
I would replace this with flexbox if this is simple for consistency with our all the others layouts